pkg/rhcos/openstack.go

```diff
@@ -2,11 +2,17 @@ package rhcos
 
 import (
 	"context"
+	"math/rand"
 	"net/url"
+	"time"
 
 	"github.com/pkg/errors"
 )
 
+const charset = "abcdefghijklmnopqrstuvwxyz"
```

Comment:
♪ ...now I know my abc's... ♪

Reply:
we can still count from two to nine :)
https://github.com/openshift/installer/blob/master/pkg/asset/password/password.go#L46

Reply:
I started with clusterID instead of a random string, but then I found that clusterID is not a dependency for the Image and BootstrapImage assets... 
https://github.com/openshift/installer/blob/master/pkg/asset/rhcos/image.go#L41
https://github.com/openshift/installer/blob/master/pkg/asset/rhcos/bootstrap_image.go#L30
So we don't know clusterID during the image name generation.

It is possible to add it there, but this slightly complicates the code and mostly useless, because the only requirement for an image is that its name be random.

Reply:
> https://github.com/openshift/installer/blob/master/pkg/asset/rhcos/image.go#L41
https://github.com/openshift/installer/blob/master/pkg/asset/rhcos/bootstrap_image.go#L30

both the assets don't control the end-image-location rather control the source of the end-image-location... so i wouldn't let these assets control the name of the image as created by the installer in Glance.
example: 

https://github.com/openshift/installer/blob/1ce819337bbdb96900942101b50205c7f69888e1/pkg/asset/machines/azure/machines.go#L96-L98 

https://github.com/openshift/installer/blob/1ce819337bbdb96900942101b50205c7f69888e1/pkg/asset/cluster/tfvars.go#L182
https://github.com/openshift/installer/blob/1ce819337bbdb96900942101b50205c7f69888e1/data/data/azure/main.tf#L157


even though there is a implicit tie between the machine object _ that uses the cloud image reference_ and the source image -> cloud image in terraform.. that's what ever other provider does, and tracking that is easier for us to fix and improve all together when we do.